service/src/main/resources/db/migration/V512__citizen_child_document_types.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TYPE document_template_type ADD VALUE 'CITIZEN_BASIC' BEFORE 'OTHER';
+ALTER TYPE child_document_status ADD VALUE 'CITIZEN_DRAFT' BEFORE 'COMPLETED';
+
+ALTER TABLE child_document
+    ADD COLUMN answered_at TIMESTAMP WITH TIME ZONE,
+    ADD COLUMN answered_by UUID REFERENCES evaka_user (id),
```

Comment:
<img width="1062" alt="Screenshot 2025-03-28 at 9 29 09" src="https://github.com/user-attachments/assets/f662f703-33f4-40a2-bc36-85c4563d0485" />

Ei tarvii vielä tehdä mitään mut mietin et jos joku lomake täytetään yhdessä huoltajan kanssa sellai että henkilökunta kirjaa vastaukset luonnostilassa ja sit klikkailee suoraan valmis-tilaan, niin pitäisköhän näihin tulla sen työntekijän tiedot ja kälis vois sit lukee tyyliin

> 28.03.2025, Lasse Lastenhoitaja (huoltajan puolesta)

Reply:
toki varmaan sit napissa pitäis kans lukee selvyyden vuoks joku "Vahvista huoltajan puolesta" tms eikä vaan julkaise valmiina

Reply:
> henkilökunta kirjaa vastaukset luonnostilassa ja sit klikkailee suoraan valmis-tilaan

tämä toiminnallisuus ei kuulu mvp-toteutukseen.